File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
angle = []
for i in range(3):
    angle.append(int(input()))

if sum(angle) != 180:
    print('Error')
else:
    if angle[0] == angle[1] and angle[1] == angle[2] and angle[2] == angle[0]:
        print('Equilateral')
    elif angle[0] != angle[1] and angle[1] != angle[2] and angle[2] != angle[0]:
        print('Scalene')
    else:
        print('Isosceles')

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.ArrayList;
import java.util.HashSet;
import java.util.List;
import java.util.Scanner;
import java.util.Set;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        List<Integer> angles =  new ArrayList<>();
        for (int i = 0; i < 3; i++) {
            int angle = sc.nextInt();
            angles.add(angle);
        }

        int sum = angles.stream()
                .mapToInt(Integer::intValue)
                .sum();

        Set<Integer> set = new HashSet<>(angles);
        
        if  (set.size() == 1 && sum == 180) {
            System.out.println("Equilateral");
        }else if (set.size() == 2 && sum == 180) {
            System.out.println("Isosceles");
        }else if (set.size() == 3 && sum == 180) {
            System.out.println("Scalene");
        } else if (sum != 180) {
            System.out.println("Error");
        }
    }
}